src/map/recast_container.h

```diff
@@ -74,6 +77,7 @@ class CRecastContainer
 
     virtual RecastList_t* GetRecastList(RECASTTYPE type);
     Recast_t*             GetRecast(RECASTTYPE type, uint16 id);
+    Recast_t*             GetRecast(LootRecastID id);
```

Comment:
The overloads look a little confusing at a glance, though this is just my opinion...

```suggestion
    bool         HasLootRecast(LootRecastID id);
    bool         HasRecast(RECASTTYPE type, uint16 id, timer::duration recast);
    virtual void Add(RECASTTYPE type, uint16 id, timer::duration duration, timer::duration chargeTime = 0s, uint8 maxCharges = 0);
    void         AddLootRecast(LootRecastID id, timer::duration duration);
    Recast_t*    Load(RECASTTYPE type, uint16 id, timer::duration duration, timer::duration chargeTime = 0s, uint8 maxCharges = 0);
    virtual void ResetAbilities();
    virtual void ChangeJob()
    {
    }

    virtual RecastList_t* GetRecastList(RECASTTYPE type);
    Recast_t*             GetRecast(RECASTTYPE type, uint16 id);
    Recast_t*             GetLootRecast(LootRecastID id);
```

Reply:
Ok, for some reason github won't render it correctly..

`bool         Has(LootRecastID id);` -> `bool         HasLootRecast(LootRecastID id);`
`void         Add(LootRecastID id, timer::duration duration);` -> `void         AddLootRecast(LootRecastID id, timer::duration duration);`
`Recast_t*             GetRecast(LootRecastID id);` -> `Recast_t*             GetLootRecast(LootRecastID id);`

Reply:
Got all those vars renamed to more descriptive versions.